include/verilated.mk.in

```diff
@@ -163,35 +157,36 @@ ifneq ($(VK_LIBS_THREADED),0)
 endif
 
 #######################################################################
-##### Stub
+### Aggregates
 
-preproc:
+VM_FAST += $(VM_CLASSES_FAST) $(VM_SUPPORT_FAST)
+VM_SLOW += $(VM_CLASSES_SLOW) $(VM_SUPPORT_SLOW)
 
 #######################################################################
-# Overall Objects Linking
+### Overall Objects Linking
 
-VK_CLASSES_FAST_CPP = $(addsuffix .cpp, $(VM_CLASSES_FAST))
-VK_CLASSES_SLOW_CPP = $(addsuffix .cpp, $(VM_CLASSES_SLOW))
-
-VK_SUPPORT_FAST_CPP = $(addsuffix .cpp, $(VM_SUPPORT_FAST))
-VK_SUPPORT_SLOW_CPP = $(addsuffix .cpp, $(VM_SUPPORT_SLOW))
```

Comment:
What you did is cleaner, but I'd prefer to keep any of the old defines, then use both VK_CLASSES/SUPPORT_FAST/SLOW below. People may have custom make files that would then break otherwise (I have a few such makefiles - though might be the only one).

I'm not worried about the ALLfast/slow being renamed ALL though.

Reply:
I appreciate that situation, I thougth that giventhese are only used under `ifneq ($(VM_PARALLEL_BUILDS),1)` they are not that significant. I'm happy to keep the old one though would love to know what you use them for externally (just curious, so don't spend time answering this if it's complicated).

Reply:
Let's cleanup then. As you had it.  Looking at the makefile I had it has other issues and this is a good excuse to fix it the right way.